content/support/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Support"
+---
+
+# I need help with PrestaShop
+
+The PrestaShop open-source project can be freely downloaded and used by anyone, but it is provided "as is", with no warranty or dedicated support. 
+
+Rest assured, there are many resources to help you if you need assistance.
+
+## Community help
+
+PrestaShop has a sizeable community, and there are many volunteers willing to help each other on the web:
+
+* [Report a bug on GitHub]({{< devdocs "contribute/contribute-reporting-issues/" >}})
+* [Join the community forums](https://www.prestashop.com/forums/)
+* [Ask on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)
+* [Discuss on Slack]({{< relref "/slack" >}})
+
+## Contact a professional
+
+If you need serious help with your project, your best choice is to get contact a professional. There are thousands of agencies and freelance experts located around the globe:
+
+* [PrestaShop partner agencies and experts](https://www.prestashop.com/experts)
```

Comment:
```suggestion
```

Reply:
I disagree with removing the link, but I think the link should be clear whether it's PrestaShop company or PrestaShop project.

```suggestion
* [PrestaShop SA partner agencies and experts](https://www.prestashop.com/experts)
```

Reply:
How about this?
```suggestion
* [List of agencies and experts](https://www.prestashop.com/experts) (sponsored by PrestaShop SA)
```

Reply:
Better 👍

Reply:
404 error you need to add `/en/` but i also agree @Progi1984